OffchainLabs/arbitrum-token-bridge
# feat: display balances for custom destination address

Closes https://github.com/OffchainLabs/arbitrum-token-bridge/issues/917

### How to test

Insert a custom destination address. It should show balances for that address in the destination network. The origin network should always show the sender wallet balances.

Also, a banner should appear in the destination network box.

Make sure the token list also displays sender's balances.

<img width="563" alt="Screenshot 2023-06-20 at 17 16 35" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/284f8902-2403-480d-86aa-8e694051e96f">
